Sources/DolbyIORTSCore/Manager/TaskScheduler.swift

```diff
@@ -12,19 +12,24 @@ protocol TaskSchedulerProtocol: AnyObject {
 }
 
 final class TaskScheduler: TaskSchedulerProtocol {
+    private static let logger = Logger(subsystem: "io.dolby.rtscore", category: String(describing: TaskScheduler.self))
```

Comment:
Shall we put this`io.dolby.rtscore` in a shared constant? It has been used 5 times in the project.

Reply:
Updated !!